python/sglang/srt/entrypoints/engine.py

```diff
@@ -472,6 +472,84 @@ def save_remote_model(self, **kwargs):
     def save_sharded_model(self, **kwargs):
         self.collective_rpc("save_sharded_model", **kwargs)
 
+    def score(
+        self,
+        text_1: str,
+        text_2: Union[str, List[str]],
+        positive_token_id: int,
+        negative_token_id: int,
```

Comment:
to make the API generic, instead of picking 1 positive and negative. Is that possible to just ask user provide a list of token ids that would like to pick up? Then you can add option to do normalize/softmax or not based the list user selected.